src/Microsoft.DotNet.Interactive/Commands/AnonymousKernelCommand.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.DotNet.Interactive.Commands
 {
-    internal class AnonymousKernelCommand : KernelCommand
+    public class AnonymousKernelCommand : KernelCommand
```

Comment:
We shouldn't make this class available as part of our public interface as it can't be serialized.